### PR TITLE
user is not allowed to create device on Thingspeak incase they are in testing ENVs

### PR DESCRIPTION
**Pull request by Baalmart**
**_WHAT DOES THIS PR DO?_**

- [x] edge case where user is not allowed to create device on Thingspeak incase they are in testing ENVs

**_HOW DO I TEST OUT THIS PR?_**
```
cd src/device-registry
npm install
npm run stage-mac
```

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

- [x] [create device](https://airqo-engineering.postman.co/workspace/internal~7a8e9fad-2d13-4d27-af27-f1f4e9a114bf/request/12513372-3be73eff-b260-4c5c-933d-5f193ee89cf5)

### CODE REVIEW

File: src/device-registry/utils/create-device.js [hunk 1]
```diff
@@ -153,9 +153,22 @@ const createDevice = {
         return {
           success: false,
           message: "creation is not yet possible for this organisation",
-          status: HTTPStatus.NOT_IMPLEMENTED,
+          status: httpStatus.NOT_IMPLEMENTED,
         };
       }
+
+      if (process.env.NODE_ENV !== "production") {
+        return {
```
Comment (Baalmart) [inline src/device-registry/utils/create-device.js:161]: in case it is not production ENV, advise the user to use SOFT creation endpoint. @FaithDaka @BenjaminSsempala
